Fix for: Using dialog to remove table header when tbody is empty throws an error (#1460)
ckeditor/ckeditor-dev: tests/plugins/table/manual/onlyrowheader.md

@bender-tags: 4.8.1, bug, table, 1397
@bender-ui: collapsed
@bender-ckeditor-plugins: wysiwygarea,toolbar,table,sourcearea,dialogadvtab,contextmenu

# Test steps

> 1. Rightclick on table, open table properties from context menu.

Reviewer: ```suggestion
1. Right click on table, open table properties from context menu.
```

Reviewer: Not sure 🤔

Author: It's not necessary to use mouse to open keystroke to reproduce this issue, so let's simplify that step:
```suggestion
1. Open context menu on table and choose table properties.
```
